app/controllers/events_controller.rb

```diff
@@ -0,0 +1,30 @@
+class EventsController < ApplicationController
+  def new; end
+
+  def index
+    @events = Event.all
+  end
+
+  def create
+    @event = current_user.events.build(description: params[:description],
+                                       event_date: params[:event_date],
+                                       location: params[:location])
+    # @event = Event.new(description: params[:description],
```

Comment:
You ought to remove all commented out code. It’s better to delete commented code. If you need to review a previous version of your code, use git history
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from Pathwright